### DISCUSSION TITLE
Share articles to external services

### DISCUSSION THREAD

**Discussion by Jei78** [martinrotter/rssguard#1425] [Q&A] [answered]
RSS Guard is seriously the best!!
I've been looking for an RSS reader client for Windows for a long time.
I love how it's the fastest and most customizable.

I have one question.
Is there a feature to share each article to external services (Instapaper, email, X, etc.)?
If not, I would be very happy if you could consider it for a future update.

**Comment by martinrotter** [comment, marked as answer]
At this point this feature is really not there. There are "external tools" feature which might be tricked to partially do what you want, but not really usefully.

In the future, this might be added.

**Reply by Jei78** [reply]
Got it. Thanks for the reply.
